add FAQ doc under 'serving' (#5946)
vllm-project/vllm: docs/source/serving/faq.rst

Frequently Asked Questions
========================

> Q: How can I serve multiple models on a single port using the OpenAI API?

Reviewer: ```suggestion
    Q: How can I serve multiple models on a single port using the OpenAI API?
```

A: Assuming that you're referring to using OpenAI compatible server to serve multiple models at once, that is not currently supported, you can run multiple instances of the server (each serving a different model) at the same time, and have another layer to route the incoming request to the correct server accordingly.

----------------------------------------

> Q: which model to use for offline inference embedding?

Reviewer: ```suggestion
    Q: Which model to use for offline inference embedding?
```

Frequently Asked Questions
========================

Q: How can I serve multiple models on a single port using the OpenAI API?

> A: Assuming that you're referring to using OpenAI compatible server to serve multiple models at once, that is not supported and not in scope of vLLM. Instead, you can run multiple instances of the server (each serving a different model) at the same time, and have another layer to route the incoming request to the correct server accordingly.

Reviewer 2: This is currently not supported. Let's not say it's out of scope yet. Thanks.

> Frequently Asked Questions
> ========================
> 
> Q: How can I serve multiple models on a single port using the OpenAI API?
> A: Assuming that you're referring to using OpenAI compatible server to serve multiple models at once, that is not currently supported, you can run multiple instances of the server (each serving a different model) at the same time, and have another layer to route the incoming request to the correct server accordingly.
> 
> Q: which model to use for offline inference embedding?
> A: If you want to use an embedding model, try: https://huggingface.co/intfloat/e5-mistral-7b-instruct. Instead models, such as Llama-3-8b, Mistral-7B-Instruct-v0.3, are generation models rather than an embedding model

Reviewer 3: I would like to suggest the following format with newlines between Q and A so they don't appear on the same line and explicit `-----` breaks between Q+A. I also added a pointer on embedding model support.
```suggestion
Frequently Asked Questions
==========================

Q: How can I serve multiple models on a single port using the OpenAI API?

A: Assuming that you're referring to using an OpenAI-compatible server to serve multiple models at once, this is not currently supported. However, you can:

- Run multiple instances of the server (each serving a different model) at the same time
- Have another layer to route the incoming requests to the correct server accordingly

----------------------------------------

Q: Which model should I use for offline inference embedding?

A: If you want to use an embedding model, try: https://huggingface.co/intfloat/e5-mistral-7b-instruct

You can view supported embedding model architectures by checking the ``_EMBEDDING_MODELS`` list in ``vllm/model_executor/models/__init__.py``
Note that models such as Llama-3-8b and Mistral-7B-Instruct-v0.3 are generation models rather than embedding models.

----------------------------------------
```

> A: If you want to use an embedding model, try: https://huggingface.co/intfloat/e5-mistral-7b-instruct. Instead models, such as Llama-3-8b, Mistral-7B-Instruct-v0.3, are generation models rather than an embedding model

Reviewer 2: I believe the there are others we are beginning to support (cc @robertgshaw2-neuralmagic
